### DISCUSSION TITLE
Copilot add semicolon al end of every function declatarion after press "Tab"

### DISCUSSION THREAD

**Discussion by hdpklm** [community/community#7740] [Projects and Issues]
Hello All

I use Copilot in vscode windows 10 - C ++ for Adruino ESP32
my problem is that every time Copilot predicts the declaration and parameters of a function. everything is fine. but it adds a semicolon at the end of the declaration as an abstract function.
Ex: 
![image](https://user-images.githubusercontent.com/35699848/141823263-d9d539a2-db43-4ab2-80c2-8f8b88b02803.png)

Thank you for letting me evaluate with Copilot, it is an unbeatable experience.
have a nice day

**Comment by sophonaroap** [comment]
I also have a similar problem. I wish I could configure copilot to never add a semicolon at the end of statements

**Reply by yarapolana** [reply]
Same! I don't use semicolons ever

**Reply by rumbledethumps** [reply]
Four years later and this is still a problem. Guess I won't be using copilot.
